includes/class-wc-moip-gateway.php~

```diff
@@ -0,0 +1,1184 @@
+<?php
```

Comment:
About the non-standard wordpress coding, what are you referring to?
Indentation?

On Tue, Jun 28, 2016 at 4:56 PM, Paulo Radicchi <paulomradicchi@gmail.com>
wrote:

> This is a auto generated "backup", it should be ignored. My bad sorry!
>
> On Thu, May 5, 2016 at 2:35 PM, Claudio Sanches <notifications@github.com>
> wrote:
>
>> In includes/class-wc-moip-gateway.php~
>> <https://github.com/claudiosmweb/woocommerce-moip/pull/15#discussion_r62222894>
>> :
>>
>> > @@ -0,0 +1,1184 @@
>> > +<?php
>>
>> What this file is doing here?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/claudiosmweb/woocommerce-moip/pull/15/files/36802d3ef3729823cb075632c7e04a01faae1903#r62222894>
>>
>
>
>
> --
> Paulo Radicchi
> paulomradicchi@gmail.com | +55 31 9287 4122
>



-- 
Paulo Radicchi
paulomradicchi@gmail.com | +55 31 9287 4122